sympy/geometry/line.py

```diff
@@ -2101,6 +2101,9 @@ def equation(self, x='x', y='y'):
         y : str, optional
             The name to use for the y-axis, default value is 'y'.
 
+        Note : Symbols being used need to be declared as real to avoid
+               unexpected results. The best approach is to provide your
+               own symbols as .equation(x, y) .
```

Comment:
I don't expect that anyone reading this would understand what point you are trying to make. A doctest example showing how to pass your own symbols and make use of them would be better.

Also probably the function should be changed so that it doesn't declare the symbols as real.

Reply:
Thanks .I'll try to improve on it !